hamilton/io/utils.py

```diff
@@ -0,0 +1,19 @@
+import os
+from datetime import datetime
+from typing import Any, Dict
+
+
+def get_file_loading_metadata(path: str) -> Dict[str, Any]:
+    """Gives metadata from loading a file.
```

Comment:
probably need to rename to `get_file_metadata`, since it's used in both save and load?